webapp/src/adapters/solid/SolidSessionManager.tsx

```diff
@@ -1,4 +1,4 @@
-import {getDefaultSession, ISessionInfo, Session} from "@inrupt/solid-client-authn-browser";
+import {getDefaultSession, Session} from "@inrupt/solid-client-authn-browser";
```

Comment:
Can you also explain this change? Thank you

Reply:
This import is no longer used